cranelift/filetests/filetests/isa/x64/tls_elf.clif

```diff
@@ -0,0 +1,19 @@
+test compile
+set tls_model=elf_gd
+target x86_64
+feature "experimental_x64"
+
+function u0:0(i32) -> i64 {
+gv0 = symbol colocated tls u1:0
+
+block0(v0: i32):
+    v1 = global_value.i64 gv0
+    return v1
+}
+
+; check:  pushq   %rbp
+; nextln: movq    %rsp, %rbp
+; nextln: elf_tls_get_addr User { namespace: 1, index: 0 }
```

Comment:
Can you add a binemit test? The exact byte sequence is important.

Reply:
Unfortunately the `binemit` filetest backend doesn't support `MachInst` yet; I'll add an issue for that.